module-audio/board/rt1051/bellpx/BellPxAudioCodec.hpp

```diff
@@ -29,6 +29,8 @@ namespace audio
 
     class BellPxAudioCodec : public SAIAudioDevice
     {
+        static constexpr auto maxVolume = 15.0f;
+        static constexpr auto minVolume = 0.0f;
```

Comment:
is this called somewhere outside the BellPxAudioCodec.cpp module? If not, please move it to an anim space inside BellPxAudioCodec.cpp. This will speed up compilation if someone modifies these values.
By the way, have you considered reducing the volume constants? Currently, there are several of them scattered in different modules

Reply:
I've reduced some of them in harmony apps. All others are used in different parts of code; some are a Pure related.

Reply:
max/min Volume are now only in two files. Both are common and Pure used them, so we need to leave the value 10 there.